.github/CODEOWNERS

```diff
@@ -129,6 +129,7 @@
 
 # E2E app
 /e2e/*                                             @jelbourn
+/e2e/components/autocomplete-e2e.spec.ts           @ann40a
```

Comment:
Can you mark crisbeto as the code owner here as he is the code owner for the component.